dashboard/app/views/levels/_contract_match.html.haml

```diff
@@ -72,6 +72,53 @@
     .clear
     = render partial: 'levels/dialog', locals: {app: app, data: data}
   .clear
+  %div{style: 'display: none;'}
+    #badname-dialogcontent
```

Comment:
I don't know if there's a good way to generate this haml in a loop (or if i should) since each of these items is the same other than body text.